smartsheet-csharp-sdk/main/Smartsheet/Api/Internal/SheetResourcesImpl.cs

```diff
@@ -298,6 +298,9 @@ public virtual PaginatedResult<Sheet> ListOrganizationSheets(PaginationParameter
             {
                 parameters.Add("page", page.ToString());
             }
+            if (rowsModifiedSince != null) {
+                parameters.Add("rowsModifiedSince", ((DateTime)rowsModifiedSince).ToUniversalTime().ToString("o"));
```

Comment:
seems like this is the format that 'o' specifies: https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier

was there a particular reason you chose this one? iso 8601? just curious

Reply:
Honestly, I was just trying to keep parity with how we do modified since in the same class. https://github.com/smartsheet/smartsheet-csharp-sdk/blob/2bc20833b8405d2518fe4258dd9cdccf73a84879/smartsheet-csharp-sdk/main/Smartsheet/Api/Internal/SheetResourcesImpl.cs#L162

Thank you for the eagle eyes though! I am glad you are making sure I made a good choice :)